komorebi/src/process_command.rs

```diff
@@ -448,6 +448,7 @@ impl WindowManager {
             SocketMessage::Retile => self.retile_all(false)?,
             SocketMessage::FlipLayout(layout_flip) => self.flip_layout(layout_flip)?,
             SocketMessage::ChangeLayout(layout) => self.change_workspace_layout_default(layout)?,
+            SocketMessage::ToggleLayout() => self.toggle_layout()?,
```

Comment:
Can you rename this to `CycleLayout`? I think `Toggle` here suggests an on/off binary state and ultimately this command will always "cycle" back to the first layout once it reaches the last one

Reply:
Done! :) Makes sense.